### PR TITLE
Revert "Adding fragment aggregation test"

### PR DESCRIPTION
**Pull request by disa6302**
This reverts commit 8e8974cad70b8a0c305ad144362c685a56463c59.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by MushMal — APPROVED** [review]
Thanks and sorry for pushing directly to master - I shouldn't have had the privileges to screw this up in the first place:).

While you don't really need to wait for this to finish building in order to push, I think it's still best to do that to make sure a clean build and the badge.

**Comment by disa6302** [comment]
> Thanks and sorry for pushing directly to master - I shouldn't have had the privileges to screw this up in the first place:).
> 
> While you don't really need to wait for this to finish building in order to push, I think it's still best to do that to make sure a clean build and the badge.

No problem Mushegh. Will wait for the CI to pass before merging.

**Pull request merged**
